front/src/modules/trainschedule/components/ManageTrainSchedule/types.ts

```diff
@@ -0,0 +1,26 @@
+import type { Position } from 'geojson';
+
+export type SuggestedOP = {
+  prId: string;
+  name?: string;
+  uic?: number;
+  ch?: string;
+  ch_long_label?: string;
+  ch_short_label?: string;
```

Comment:
Should we use CamelCase ? 
```suggestion
  chLongLabel?: string;
  chShortLabel?: string;
```

Reply:
More consistent with the other properties, you're right.

Reply:
So we should do the same for stop_for > stopFor

Reply:
Yes, thanks :)